src/Xamarin.Android.Build.Tasks/Microsoft.Android.Sdk/targets/Microsoft.Android.Sdk.targets

```diff
@@ -27,6 +27,10 @@
     <ProjectCapability Include="Mobile" />
     <ProjectCapability Include="Android" />
     <ProjectCapability Include="AndroidApplication" Condition="$(AndroidApplication)" />
+    
+    // See https://work.azdo.io/1112733
+    <!-- Conflicts with our targets generator in VS+CPS -->
+    <ProjectCapability Remove="LaunchProfiles" />
```

Comment:
```suggestion
    <!-- Conflicts with our targets generator in VS+CPS, see https://work.azdo.io/11127 -->
    <ProjectCapability Remove="LaunchProfiles" />
```
It's weird that raw text content was valid XML.

Reply:
it wasn't :(